website/docs/cloud-docs/api-docs/run-tasks/run-task-stages-and-results.mdx

```diff
@@ -0,0 +1,259 @@
+        "running-at": "2022-06-08T20:32:11+08:00"
+      },
+      "created-at": "2022-06-08T12:31:56.94Z",
+      "updated-at": "2022-06-08T12:32:12.315Z"
+    },
+    "relationships": {
+      "run": {
+        "data": {
+          "id": "run-XdgtChJuuUwLoSmw",
+          "type": "runs"
+        }
+      },
+      "task-results": {
+        "data": [
+          {
+            "id": "taskrs-EmnmsEDL1jgd1GTP",
+            "type": "task-results"
+          }
+        ]
+      }
+    },
+    "links": {
+      "self": "/api/v2/task-stages/ts-rL5ZsuwfjqfPJcdi"
+    }
+  }
+}
+```
+
+## Show a Task Result
+
+`GET /task-results/:task_result_id`
+
+| Parameter         | Description                |
+|-------------------|----------------------------|
+| `:task_result_id` | The task result ID to get. |
+
+This endpoint is used for showing details of a specific task result.
+
+| Status  | Response                                       | Reason                                       |
+|---------|------------------------------------------------|----------------------------------------------|
+| [200][] | [JSON API document][] (`type: "task-results"`) | Success                                      |
+| [404][] | [JSON API error object][]                      | Task result not found or user not authorized |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/task-results/taskrs-EmnmsEDL1jgd1GZz
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "taskrs-EmnmsEDL1jgd1GZz",
+    "type": "task-results",
+    "attributes": {
+      "message": "No issues found.\nSeverity threshold is set to low.",
+      "status": "passed",
+      "status-timestamps": {
+        "passed-at": "2022-06-08T20:32:12+08:00",
+        "running-at": "2022-06-08T20:32:11+08:00"
+      },
+      "url": "https://external.service/project/task-123abc",
+      "created-at": "2022-06-08T12:31:56.954Z",
+      "updated-at": "2022-06-08T12:32:12.27Z",
+      "task-id": "task-b6MaHZmGopHDtqhn",
+      "task-name": "example-task",
+      "task-url": "https://external.service/task-123abc",
+      "stage": "post_plan",
+      "is-speculative": false,
+      "workspace-task-id": "wstask-258juqenQeWb3DZz",
+      "workspace-task-enforcement-level": "mandatory"
+    },
+    "relationships": {
+      "task-stage": {
+        "data": {
+          "id": "ts-rL5ZsuwfjqfPJczZ",
+          "type": "task-stages"
+        }
+      }
+    },
+    "links": {
+      "self": "/api/v2/task-results/taskrs-EmnmsEDL1jgd1GZz"
+    }
+  }
+}
+```
+
+## Available Related Resources
+
+### Task Stage
+The GET endpoints above can optionally return related resources, if requested with [the `include` query parameter](/cloud-docs/api-docs/#inclusion-of-related-resources). The following resource types are available:
+
+- `run` - Additional information about the associated run.
```

Comment:
Nit: we use tables to display the states for Task Stages and Task Results, but a list for this. Should we use table here for consistency?

Reply:
Hrmm... not saying it's right but this is format in other documents like run. Could probably be a table. I'll change it.